File: frontend/hustle-app/src/app/models/Event.ts
import { Venue } from "./venue";
import { Sport } from "./sport";
import { Image } from "./image";
export class Event {
   id:number;
   name : string;
   description:string;
   venue: Venue;
   sport:Sport;
   image:Image;
   date :string;
   constructor(name:string,description:string,
                venue:Venue,sport:Sport,image:Image,date:string) {
                  this.name = name;
                  this.description = description;
                  this.venue = venue;
                  this.sport = sport;
                  this.image = image;
                  this.date=date;
                }
}

File: backend/src/main/kotlin/hr/fer/ppij/model/Event.kt
package hr.fer.ppij.model

import javax.persistence.*

@Entity
@Table(name = "events")
data class Event (
        @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Long? = null,
        var name: String,
        var description: String,
        @ManyToOne
        @JoinColumn(name = "venue_id")
        var venue: Venue,
        @ManyToOne
        @JoinColumn(name = "sport_id")
        var sport: Sport,
        @ManyToOne
        @JoinColumn(name = "image_id")
        var image: Image? = null
)